script.js

```diff
@@ -0,0 +1,30 @@
+function calculateAnswer () {
+    
+    let firstNumberElement = document.getElementById("firstNumber")
+    let secondNumberElement = document.getElementById("secondNumber")
+
+    let firstNumberValue = Number(firstNumberElement.value)
+    let secondNumberValue = Number(secondNumberElement.value)
```

Comment:
Technically, you can combine lines 6 & 7 with lines 3 & 4 by adding the .value to the end of lines 3 & 4 and wrapping the whole thing in the Number() method. But, breaking it down the way you have it can sometimes help with readability! Either way is valid.